tests/test_json_schema.py

```diff
@@ -3459,3 +3459,46 @@ class OuterModel(BaseModel):
         'title': 'OuterModel',
         'type': 'object',
     }
+
+
+def test_multi_host_url_schema():
+    class TestGenerateJsonSchema(GenerateJsonSchema):
```

Comment:
Why do we need this class here?

Reply:
@hramezani   Removed the unneeded class in the 3 tests.